src/vlinder/presentation.py

```diff
@@ -0,0 +1,43 @@
+from fpdf import FPDF
+
+
+class PDF(FPDF):
```

Comment:
Can we move this class to make_report.py s.t. we have all the report material in the same file?

Reply:
Unfortunately hard to do because of some specific class attributes in PDF(FPDF)